docker-compose.override.yml

```
@@ -43,6 +61,19 @@ services:
   # --- User Service ---
   # For local development
 
+  # in the development environment we want the user service to depend on the key-gen, because
+  # without it JWTs can't be signed and validated. Also, we are defining volumes and env vars.
+  # which the user service will need only in local dev. environment
+  user-service:
+    volumes:
+      - ./keys:/etc/spotlite/keys:ro
+    environment:
+      JWT_PRIVATE_KEY_PATH: /etc/spotlite/keys/private.pem
+      JWT_PUBLIC_KEY_PATH: /etc/spotlite/keys/public.pem
```

I've seen `/run` be more advisable for runtime data. `/etc` on Linux-based systems by standard are used for permanent static configurations, while `/run` is used for generated, boot session files for dynamic use.

This is also what [Docker secrets](https://docs.docker.com/engine/swarm/secrets/#how-docker-manages-secrets) use.

```suggestion
  user-service:
    volumes:
      - ./keys:/run/keys:ro
    environment:
      JWT_PRIVATE_KEY_PATH: /run/keys/private.pem
      JWT_PUBLIC_KEY_PATH: /run/keys/public.pem
```

Nice work on the `read-only` flag ^^